src/spikeinterface/extractors/tests/test_neoextractors.py

```diff
@@ -290,6 +290,32 @@ def test_pickling(self):
         pass
 
 
+# We mark plexon2 tests as they require additional dependencies (wine)
+@pytest.mark.plexon2
```

Comment:
In POSIX the dependencies are wine and https://pypi.org/project/zugbruecke/ right?

In windows what are the dependecies?

I don't like that if I don't have wine or this experimental packages my test will fail if run pytest -m "extractors"

Can we add skipif tests to accomplish this? (this can be another PR if you think this is too much)

I am suggesting:

```python
import subprocess
import platform

def has_dependencies():
    """
    Check if required dependencies are installed.
    """
    
    os_type = platform.system()

    if os_type == "Windows":
        # On Windows, check for dependencies if any (probably just return True)
        return True

    elif os_type == "Linux":
        # Check for 'wine' using dpkg
        try:
            result_wine = subprocess.run(['dpkg', '-l', 'wine'], 
                                         stdout=subprocess.PIPE, stderr=subprocess.PIPE, check=True)
        except subprocess.CalledProcessError:
            return False
        
        # Check for 'zugbruecke' using pip
        try:
            import zugbruecke
            return True
        except subprocess.CalledProcessError:
            return False

    else:
        raise ValueError(f"Unsupported OS: {os_type}")

# Use pytest.mark.skipif to conditionally skip the test class
@pytest.mark.skipif(not has_dependencies(), reason="Required dependencies not installed")
@pytest.mark.plexon2
class Plexon2RecordingTest(RecordingCommonTestSuite, unittest.TestCase):
    ExtractorClass = Plexon2RecordingExtractor
    downloads = ["plexon"]
    entities = [
        ("plexon/4chDemoPL2.pl2", {"stream_id": "3"}),
    ]
        ```

Reply:
I also think that maybe this will help us avoid adding -some- complexity to the testing in the CI but less sure about it.

Reply:
Good point @h-mayorquin , I'll do something along these lines!

Reply:
Done!